lib/find-visualstudio.js

```diff
@@ -113,9 +114,25 @@ class VisualStudioFinder {
     throw new Error('Could not find any Visual Studio installation to use')
   }
 
+  // Invoke the PowerShell script to get information about Visual Studio 2019
+  // or newer installations
+  async findVisualStudio2019OrNewer () {
+    return this.findNewVS([2019, 2022])
+  }
+
+  // Invoke the PowerShell script to get information about Visual Studio 2017
+  async findVisualStudio2017 () {
```

Comment:
This function should be removed. Also this should be a breaking change. commit. msg should be like 
```
win!: update supported vs versions
```

Reply:
I've changed the commit message. Should I also add the `semver-major` label?

Regarding the removal of `findVisualStudio2017`, why do you think it should be removed? I've added it in this PR so that VS2017 can be prohibited on Node v22+, while VS2019 and VS2022 will be enabled. On previous Node versions all 3 VS versions should work, like they are working in the current `main` branch.

Reply:
I don't follow the reasoning why this would be a breaking change. @gengjiawen Can you elaborate on this?